modules/vaos/lib/vaos.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'vaos/engine'
+
+module VAOS
+  # Your code goes here...
```

Comment:
Is this a placeholder?

Reply:
yes, also from the generator, `require 'vaos/engine'` is needed but I'll remove the empty module